source/index.html

```diff
@@ -1,12 +1,151 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
+    <link href="css/style.css" rel="stylesheet">
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS, уровень 2</a>».</p>
-
+  <body class="page__body">
+    <div class="container">
+      <header class="page-header">
+        <a class="page-header__logo logo" href="">
+        <img class="logo__img" src="" width="" height="" alt="Логотип проект Седона">
+        </a>
+        <nav class="main-nav">
+          <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
```

Comment:
```suggestion
          <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню.</span></button>
```

Добавь точку в конце, тк скринридер на точках делает паузы